packages/typespec-ts/src/lib.ts

```diff
@@ -71,7 +71,8 @@ export const RLCOptionsSchema: JSONSchemaType<RLCOptions> = {
     },
     azureArm: { type: "boolean", nullable: true },
     sourceFrom: { type: "string", nullable: true },
-    isModularLibrary: { type: "boolean", nullable: true, default: false }
+    isModularLibrary: { type: "boolean", nullable: true, default: false },
+    enableOperationGroup: { type: "boolean", nullable: true }
```

Comment:
this is boolean, not sure where can you get a string ?

Reply:
There is a bug in compiler, https://github.com/microsoft/typespec/issues/2148